src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -350,8 +357,10 @@ class ContentSidebar extends React.Component<Props, State> {
                             metadataSidebarProps={metadataSidebarProps}
                             onVersionChange={onVersionChange}
                             onVersionHistoryClick={onVersionHistoryClick}
-                            refreshIdentity={refreshIdentity}
                             versionsSidebarProps={versionsSidebarProps}
+                            wrappedComponentRef={ref => {
```

Comment:
`<Sidebar />` is wrapped in `withRouter` HOC, so we use the `wrappedComponentRef` prop to pass the ref through to the wrapped component. It expects a function: https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md#wrappedcomponentref-func